src/components/Card/Card.scss

```diff
@@ -12,6 +12,13 @@
   @include page-content-when-not-fully-condensed {
     border-radius: var(--p-border-radius-wide, border-radius());
   }
+
+  &.newDesignLanguage {
```

Comment:
Nitpick: Thoughts on moving this up  to be closer to the margin-top override above?

Not tested this but something like:

```
.Card {
  + .Card {
    margin-top: spacing(loose);

    &.newDesignLanguage {
      margin-top: spacing()
    }
  }
}
```